theseus/geometry/se3.py

```diff
@@ -622,6 +623,13 @@ def transform_to(
 
         return ret
 
+    # The quaternion takes the [w x y z] convention
+    def to_x_y_z_quaternion(self) -> torch.Tensor:
```

Comment:
Also should we provide jacobian for this operation?

Reply:
> Convention of w x y z can be confusing with the name `to_x_y_z_quaternion`.
> 
> Maybe use `to_quaternion_position` especially if we are standardizing to rotation-then-translation convention then.

This is consistent with `SE2.to_x_y_theta()`. I'm OK to rename these function names.

Reply:
> Also should we provide jacobian for this operation?

Definitely we can add it. However, the jacobian of the `SO3->unit quaterion` is rarely used and a little complicated since unit quaternions are not on the Euclidean space.

Reply:
More explicit comments added.